test/test_other.py

```diff
@@ -3739,6 +3740,25 @@ def clean(txt):
 
     self.assertTextDataIdentical(clean(proc.stdout), clean(proc2.stdout))
 
+  def test_file_packager_response_file(self):
+    filenames = set()
```

Comment:
Just use a list here?   Keeps the order deterministic and I don't think we use any set features below?

Reply:
Oh wait, just in case they collide I guess.. seems fine in that case.